packages/gitbook/src/lib/api.ts

```diff
@@ -818,7 +818,7 @@ export async function getCurrentSiteLayoutData(args: {
 export async function getCurrentSiteCustomization(args: {
     organizationId: string;
     siteId: string;
-    siteSpaceId: string | undefined;
+    siteSpaceId?: string;
```

Comment:
Same comment about making it optional

Reply:
It's just a syntactical change to make it more standard. This is already optional as it is typed as `string | undefined`. If we don't want it to be optional we should remove undefined from the type.

Reply:
Yep `string | null` should be good